file_paths.md

```diff
@@ -339,6 +340,39 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# pxp-agent
+
+This package executes Puppet and replies to PXP requests.
+
+## *nix
+
+    /opt/puppetlabs/pxp-agent *
+        spool                             # :spooldir
+        modules *                         # :modulesdir
+            puppet *                      # executes Puppet and provides results of Puppet runs
+
+    /var/log/puppetlabs/pxp-agent *       # :logdir
```

Comment:
Nevermind, the font was telling me this was uppercase. Ugh. Sorry. Nothing to see here.